.github/workflows/step-run-tests.yml

```
@@ -156,3 +156,14 @@ jobs:
         report_paths: report.xml
         fail_on_failure: true
         require_passed_tests: true
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)